src/Sentry.Maui.CommunityToolkitMvvm/CtMvvmMauiElementEventBinder.cs

```diff
@@ -0,0 +1,151 @@
+                TryBindTo(refresh.Command, bind);
+                break;
+
+            case SearchBar searchBar:
+                TryBindTo(searchBar.SearchCommand, bind);
+                break;
+
+            default:
+                TryGestureBinding(element, bind);
+                break;
+        }
+    }
+
+    private static void TryGestureBinding(VisualElement element, bool bind)
+    {
+        if (element is IGestureRecognizers gestureRecognizers)
+        {
+            foreach (var gestureRecognizer in gestureRecognizers.GestureRecognizers)
+            {
+                TryBindTo(gestureRecognizer, bind);
+            }
+        }
+    }
+
+    private static void TryBindTo(IGestureRecognizer recognizer, bool bind)
+    {
+        switch (recognizer)
+        {
+            case TapGestureRecognizer tap:
+                TryBindTo(tap.Command, bind);
+                break;
+
+            case SwipeGestureRecognizer swipe:
+                TryBindTo(swipe.Command, bind);
+                break;
+
+            // no commands
+            //case PinchGestureRecognizer pinch
+            //case PanGestureRecognizer pan
+            case DragGestureRecognizer drag:
+                TryBindTo(drag.DragStartingCommand, bind); // unlikely to ever be async
+                TryBindTo(drag.DropCompletedCommand, bind);
+                break;
+
+            case PointerGestureRecognizer pointer:
+                TryBindTo(pointer.PointerPressedCommand, bind);
+                TryBindTo(pointer.PointerReleasedCommand, bind);
+                TryBindTo(pointer.PointerEnteredCommand, bind); // unlikely to ever be async
+                TryBindTo(pointer.PointerExitedCommand, bind);
+                break;
+        }
+    }
+
+    private static List<IAsyncRelayCommand> _refs = [];
+    private static void TryBindTo(ICommand? command, bool bind)
+    {
+        if (command is IAsyncRelayCommand relayCommand)
+        {
+            if (bind)
+            {
+                // necessary for collectionview buttons
+                if (!_refs.Contains(relayCommand))
+                {
+                    _refs.Add(relayCommand);
+                    relayCommand.PropertyChanged += RelayCommandOnPropertyChanged;
+                }
+            }
+            else
+            {
+                _refs.Remove(relayCommand);
+                relayCommand.PropertyChanged -= RelayCommandOnPropertyChanged;
+            }
+        }
+    }
+
+    private static ConcurrentDictionary<IAsyncRelayCommand, (ITransactionTracer Transaction, ISpan Span)> _contexts = new();
+
+    private static void RelayCommandOnPropertyChanged(object? sender, PropertyChangedEventArgs e)
+    {
+        if (e.PropertyName == nameof(IAsyncRelayCommand.IsRunning))
+        {
+            var relay = (IAsyncRelayCommand)sender!;
+
+            if (relay.IsRunning)
+            {
+                var transaction = SentrySdk.StartTransaction("ctmvvm", "asynccommand");
```

Comment:
Could the name of the command be used as transaction name instead? 

Something like:

```suggestion
                var transaction = SentrySdk.StartTransaction(relay.CommandName, "async.command");
```

The UI will group the transactions based on name. This way you can compare the duration and operations under each command separately

Reply:
There is no command name to use.  I was looking.  I was thinking of looking up the binding it is pointed at, but that comes at a higher cost than I would like.  Even if I do get the binding, very few people add x:Name to the control, so I have to do reflection to see what the actual property name is on the viewmodel (which also kills AOT).  It's a rock & hard place.... was hoping to start this as an MVP and build on it from there.

Reply:
There is nothing called name or like name on the async command implementation or interfaces.  Retrieving the binding info from the control is not pretty and requires reflection.  With gesture recognizers (coming soon) and all of the other events potentially auto-breadcrumbing in the future, the user should have a pretty good idea of what's going on.  I can pass through the control type & control name (if set), but that also comes with some costs.  

I really think we should start with an MVP and let users decide what/if anything is missing.